cmd/embedded-cluster/update.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/defaults"
+	"github.com/replicatedhq/embedded-cluster/pkg/kotscli"
+	"github.com/replicatedhq/embedded-cluster/pkg/release"
+)
+
+var updateCommand = &cli.Command{
+	Name:  "update",
+	Usage: fmt.Sprintf("Update %s", binName),
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:   "airgap-bundle",
+			Usage:  "Path to the airgap bundle.",
+			Hidden: true,
+		},
+	},
```

Comment:
IMO make the entire command hidden for now, but make the airgap-bundle flag required